sql/character.anticheat.sql

```diff
@@ -0,0 +1,35 @@
+DROP TABLE IF EXISTS `daily_players_reports`;
```

Comment:
since it crashed for me without import this, imho it needs conversion script / alter for older version.

Reply:
U added new collumns so i had re-include the drop

Reply:
ok. i think i got it right after @Jinnaix re-explained what u ment to me. please look it over.